docker-compose.yml

```diff
@@ -8,21 +8,31 @@
 version: "3.9"
 
 services:
+  start_app_store:
+    container_name: start_app_store
+    image: jwilder/dockerize
+    command: >
+      -wait tcp://nscc-appstore:3000
+      -wait-retry-interval 10s
+      -timeout 180s
+    depends_on:
+      - nscc-db-services
+      - nscc-appstore
+
   start_applications:
     container_name: start_applications
     image: jwilder/dockerize
     command: >
       -wait tcp://nscc-provider:3000
       -wait tcp://nscc-caseworker:3000
-      -wait tcp://nscc-appstore:8000
       -wait-retry-interval 10s
       -timeout 180s
     depends_on:
       - nscc-db-services
```

Comment:
do you need this in here if it is in `start_app_store`? I often found taking things out didn't actually work but would be interested to know.

Reply:
I've removed it, and it's still working!